kdf.h

```diff
@@ -8,5 +8,6 @@
 #define KDF_HEX_LEN (KDF_HASH_LEN * 2 + 1)
 void kdf_login_key(const char *username, const char *password, int iterations, char hex[KDF_HEX_LEN]);
 void kdf_decryption_key(const char *username, const char *password, int iterations, unsigned char hash[KDF_HASH_LEN]);
+void fail_invalid_iteration_count();
```

Comment:
This function does not need to be exposed on the public API of `kdf.h`, because its only use is internally within the implementation of `kdf.c`.